src/main/java/wooteco/subway/service/PathService.java

```diff
@@ -23,12 +24,12 @@ public PathService(final StationRepository stationRepository, final SectionRepos
     }
 
     @Transactional(readOnly = true)
-    public PathResponse findPath(final Long sourceId, final Long targetId) {
+    public PathResponse findPath(final FindPathRequest findPathRequest) {
```

Comment:
```suggestion
    public PathResponse findPath(final FindPathRequest request) {
```

하나의 요청 DTO만 받는 경우에는 그냥 request로 표현해도 충분히 이해할 수 있다고 생각합니다😉

Reply:
동의합니다! 변경했어요👍🏻